core/src/main/resources/license-mapping.json

```diff
@@ -51,31 +55,36 @@
         "https://javaee.github.io/javamail/LICENSE",
         "https://oss.oracle.com/licenses/CDDL+GPL-1.1"
     ],
-    "GPL-2.0": [
+    "EPL-1.0 OR MPL-2.0": [
+        "https://h2database.com/html/license.html"
+    ],
+    "GPL-2.0-only": [
         "http://www.gnu.org/copyleft/gpl.html",
         "http://www.gnu.org/licenses/gpl.txt"
     ],
     "GPL-2.0 WITH Universal-FOSS-exception-1.0": [
         "The GNU General Public License, v2 with FOSS exception",
         "The GNU General Public License, v2 with Universal FOSS Exception, v1.0"
     ],
-    "GPL-2.0-with-classpath-exception": [
+    "GPL-2.0-only WITH Classpath-exception-2.0": [
```

Comment:
please add "https://projects.eclipse.org/license/secondary-gpl-2.0-cp" to this list

Reply:
I can add these.

Just to confirm, all of these URLs reported a missing mapping with with the current code (meaning that it was unable to match automatically)?

Reply:
Yes those URLs did not match correct licenses, the above matched `GPL-2.0-or-later` instead of `GPL-2.0-only WITH Classpath-exception-2.0`. The 2 below matched `LGPL-2.1-or-later` instead of `LGPL-2.1-only`.